activities/recruitment/company-description.md

```diff
@@ -0,0 +1,43 @@
+---
+type: resource
+---
+
+# Company description for job advertising websites
+
+> Written initially for Stack Overflow (using their subheadings), this can be reused or remixed elsewhere.
+
+## About Foundation for Public Code
+
+We aim to build international public sector communities working together to share open source software and policy in a radically collaborative and open way. Public organizations are building more open software than ever before, but collaborating at the scale needed to co-develop and maintain a codebase is still hard - it's not the core responsibility of any single public organization.
+
+The Foundation for Public Code was set up to plug this infrastructure gap so that people already working for the public good can more easily:
+
+* Find transformative codebases to reuse
+* Design their code in a way that encourages international reuse
+* Build thriving open source communities around their codebases
+* Get institutional support in navigating the risk and procurement hurdles to using open source for public services and infrastructure
+
+We do this through active codebase stewardship and by publishing the [Standard for Public Code](https://standard.publiccode.net/).
+
+### How we work
+
+All of us fundamentally believe that our public organizations using FOSS will result in better public services for everyone and be a better use of taxpayer money. We do impactful work as part of a mission we believe in.
+
+We're small, open, flexible and non-hierarchical. We act with integrity and compassion towards each other and the broader communities we're a part of. We make decisions by rough consensus, and value the diversity of perspectives on our team.
+
+We hire the best people we can, trust their expertise, and give them near infinite autonomy. We also support each other, and do lots of pair working.
+
+Because nobody has ever done this before for government and we've just started, everything we're doing is greenfield - everyone here comes to work every day knowing that what they'll do that day will meaningfully shape the Foundation's future.
+
+We're ambitious but humble - we sincerely want this to succeed, but there's no guarantee. Because we believe that failure is the best teacher, we encourage curiosity, experimentation and learning.
+
+### Benefits
+
+* 25 days of paid leave plus an average of 6 public holidays
+* enthusiastic BYOD culture
```

Comment:
I think BYOD should also be explained to make the text more accessible

Reply:
done!